test/cbmc/.gitignore

```diff
@@ -12,5 +12,12 @@ arpa_cmake/
 arpa-validation-logs/
 Makefile.arpa
 
-# Generated by litani
+# Emitted by litani
+.ninja_deps
+.ninja_log
 .litani_cache_dir
+
+# These files should be overwritten whenever prepare.py runs
```

Comment:
I noticed you delete prepare.py, is this still necessary?

Reply:
No, I've removed it.